index.html

```diff
@@ -219,7 +228,7 @@ <h1 class="primary">About Me</h1>
       </div>
       <div class="img-aboutbox">
         <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=" Diagram illustration 4"></section>
-        <!-- <img src="./images/illustration-about me@2x.png" alt="Diagram illustration 4 mobile version" class="img-aboutbig"> -->
+        <img src="./images/illustration-about me@2x.png" alt="Diagram illustration 4 mobile version" class="img-aboutbig">
       </div>
     </div>
     <div class="divi"></div>
```

Comment:
- [ ] Please make sure our skills section matches the figma template design, currently it is aligned to the left please make sure it is centered the line and the skills section your design :point_down: 

![image](https://user-images.githubusercontent.com/97521557/206562896-b928906c-13b4-41b5-909a-2d9fa5b55568.png)

Figma design :point_down: 

![image](https://user-images.githubusercontent.com/97521557/206563032-f642cad7-0636-4fa0-909f-fbcc0e5ad89b.png)